azure-resources/ArtificialIntelligence/AI/recommendations.yaml

```diff
@@ -0,0 +1,51 @@
+- description: Ensure to deploy a pay as you go instance of the model when using provisioned throughput to manage overflow effectively
+  aprlGuid: 0c193899-da60-4a52-b4a0-77d75ac8c5c5
+  recommendationTypeId: null
+  recommendationControl: Resiliency
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```

Comment:
The value needs to be a valid resource type and should align with the directory within which the recommendation is located.

Reply:
Agree. Changes pushed.

Reply:
Agree. Changes pushed.